examples/vue-router/src/ReloadPrompt.vue

```diff
@@ -25,27 +29,42 @@ const {
       console.log(`SW Registered: ${r}`)
     }
   },
+  onBeginUpdate() {
+    console.log('Begin sw update message')
+    beginUpdateServiceWorker.value = true
+    lastBeginUpdateServiceWorkerTime.value = Date.now()
+  },
+})
+
+watch(computed(() => [offlineReady.value, needRefresh.value]), ([offlineReadyVal, needRefreshVal]) => {
```

Comment:
why computed here? `watch(() => [offlineReady.value, needRefresh.value], ([offlineReadyVal, needRefreshVal]) => {`

Reply:
For reactive with both offlineReady and needRefresh.
I guess maybe we shouldn't care about offineReady here?

Reply:
`watch` isn't reactive?

Reply:
Oh, I don't need to use computed here, just a pure function is OK

Reply:
Solved